src/pattern-library/components/patterns/data/DataTablePage.tsx

```diff
@@ -1,17 +1,104 @@
-import { useState } from 'preact/hooks';
+import { useCallback, useMemo, useRef, useState } from 'preact/hooks';
 
-import { DataTable, Scroll } from '../../../../';
+import { DataTable, type DataTableProps, Scroll } from '../../../../';
+import type { Order } from '../../../../components/data/DataTable';
 import Library from '../../Library';
 import { nabokovNovels } from '../samples';
 import type { NabokovNovel } from '../samples';
 
 const nabokovRows = nabokovNovels();
 const nabokovColumns = [
-  { field: 'title', label: 'Title' },
-  { field: 'year', label: 'Year' },
-  { field: 'language', label: 'Language' },
+  { field: 'title' as const, label: 'Title' },
+  { field: 'year' as const, label: 'Year' },
+  { field: 'language' as const, label: 'Language' },
 ];
 
+type SimpleNabokovNovel = Omit<NabokovNovel, 'translatedTitle'>;
```

Comment:
I think this name is better. "Strict" sounded like it was something to do with TypeScript strict-ness.